datadog/1.11/README.md

```diff
@@ -0,0 +1,207 @@
+
+![Host map showing DC/OS nodes in Datadog](img/dcos-host-map.png)
+
+You'll also see metrics from your cluster flowing into the [Mesos dashboard][mesos-dash] and the [Docker dashboard][docker-dash] in Datadog, so you can immediately start to track your cluster resources and how they are being consumed. Similarly, you will see [ZooKeeper metrics][zk-dash] flowing into Datadog from your master nodes.
+
+You can then build custom dashboards using any of the metrics or events being reported to Datadog; set alerts on performance metrics or task failures; or use the host map to look for hotspots in your infrastructure.
+
+## Use Autodiscovery to monitor additional services
+
+### Automatically monitored services
+
+When you install the Datadog package on DC/OS, or use the command above to install Datadog on your master nodes, [Autodiscovery][autodiscovery] is enabled by default. This feature automatically detects which containerized services are running on which nodes, and configures the Datadog Agent on each node to connect to those services and report metrics back to Datadog.
+
+For certain services, such as Redis, Elasticsearch, and Apache server (httpd), the Datadog Agent can connect without any additional setup in most cases. (See the [Autodiscovery docs][autodiscovery] for the full list.) If you're running one of those services, you should start seeing metrics in Datadog within minutes (on [the Redis dashboard][redis-dash], for instance).
+
+### Monitoring services with custom configurations
+
+For other services, you can create monitoring configuration templates that Datadog will apply whenever it detects the service running on a node in your cluster. For production use, you will likely want to [use a key-value store][autodiscovery] such as etcd or Consul to manage these templates for you. To demonstrate the basics of service discovery, though, here we'll walk through a simple example using configuration templates stored in a directory on the host that is mounted by the Datadog Agent container.
+
+#### Create a config template
+
+On your agent nodes, create a directory to house your custom configs and create a YAML file for the service you want to monitor (drawing on [the example YAML templates][yamls] that ship with the Datadog Agent). In this example we'll set up service discovery for MYSQL, which requires a password to access database metrics:
+
+```bash
+# get a node ID
+dcos node
+# connect to a node selected from the previous command's output
+dcos node ssh --master-proxy --mesos-id=225ca9ef-f1bc-43c1-bf31-b9330f89bb50-S0
+# create a config folder for the mysql integration
+mkdir -p /opt/datadog-agent-conf.d/mysql.d/
```

Comment:
Got this right in the 1.10 instructions, but you do need "sudo" here for 1.11 too (and in the next line, below).